src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -76,6 +81,9 @@ class HearingsDetailsMappingTest extends HearingsMappingBase {
     private VenueService venueService;
 
     public static final String PROCESSING_VENUE_1 = "test_place";
+    private static final String PHONE_NUMBER = "07483871426";
+    private static final String HEARING_VENUE_ID_CURRENT = "219164";
+    private static final String HEARING_VENUE_ID_OTHER = "436578";
 
     private String epimsId1;
```

Comment:
Can I suggest that these epims ids are made constants instead of set in setup and are used instead of HEARING_VENUE_ID_CURRENT and HEARING_VENUE_ID_OTHER,

Reply:
Done